docs/approach.rst

```diff
@@ -235,23 +236,23 @@ in a dimensionally reduced version of the dataset which is then used in the `TED
 
 TEDICA
 ``````
-Next, ``tedana`` applies TE-dependent independent components analysis (ICA) in
+Next, ``tedana`` applies TE-dependent independent component analysis (ICA) in
 order to identify and remove TE-independent (i.e., non-BOLD noise) components.
 The dimensionally reduced optimally combined data are first subjected to ICA in
-order to fit a mixing matrix to the whitened data. 
-This generates a number of 
-independent timeseries (saved as **meica_mix.1D**), as well as beta maps which show 
-the spatial loading of these components on the brain (**betas_OC.nii.gz**). 
+order to fit a mixing matrix to the whitened data.
+This generates a number of
+independent timeseries (saved as **meica_mix.1D**), as well as beta maps which show
+the spatial loading of these components on the brain (**betas_OC.nii.gz**).
 
 .. image:: /_static/a13_ica_component_timeseries.png
 
 Linear regression is used to fit the component time series to each voxel in each
-of the original, echo-specific data. 
-This results in echo- and voxel-specific 
+of the original, echo-specific data.
+This results in echo- and voxel-specific
 betas for each of the components.
-The beta values from the linear regression 
-can be used to determine how the fluctutations (in each component timeseries) change 
-across the echo times. 
+The beta values from the linear regression
+can be used to determine how the fluctuations (in each component timeseries) change
+across the echo times.
 
 TE-dependence (:math:`R_2` or :math:`1/T_{2}^*`) and TE-independence (:math:`S_0`) models can then
 be fit to these betas.
```

Comment:
:ref:`dependence models` does not seem to be working in the compiled file. I know we had a similar issue before & it worked on the actual RTD site, but the other :ref: links seem to be working and this one does not. Not sure if something changed in this revision or an earlier revision, or if it will compile correctly on the RTD site.

Reply:
I can fix this with a hidden toctree and a new parameter in the html configuration in conf.py.

Reply:
At least in my local running of <make html> the link to dependence models is still not working.

Reply:
It's probably a Sphinx version issue. The `:hidden:` directive and `:includehidden:` parameter were both added in like 1.4 I think. Tedana's current requirements use a later version than that and the RTD CI build didn't have any warnings (just checked) so if you update your version that might help.

Reply:
Not sure what the issue is, but the RTD page that's outputted from the CI  documentation creation that's still pending looks fine so this is resolved.